src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const left = Big(numberOne);
+  const right = Big(numberTwo);
+  let result = 0;
+
+  switch (operation) {
+    case '+':
+      result = `${left.plus(right)}`;
+      break;
+    case '-':
+      result = `${left.minus(right)}`;
+      break;
+    case 'x':
+    case 'X':
+      result = `${left.times(right)}`;
+      break;
+    case '÷':
+    case '/':
+      result = right === 0 ? 'Divided by zero Error' : `${left.div(right)}`;
+      break;
+    case '%':
+      result = `${left.mod(right)}`;
+      break;
+    default:
```

Comment:
You already declare a variable, you don't need `$` to use it.
And also I think we could improve the naming conversion, `num1` instead of `left`.

Example:

      result = num1.plus(num2);

Reply:
Changed both variable names. And the $ is for template strings, I need a variable to be treated as a string, so I can't see your point